example/originv3/entry_management/entry_management.go

```diff
@@ -0,0 +1,171 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	originv3 "github.com/EdgeCast/ec-sdk-go/edgecast/originv3"
+	"github.com/kr/pretty"
+)
+
+func main() {
+
+	// Setup
+	apiToken := "MY_API_TOKEN"
+
+	idsCredentials := edgecast.IDSCredentials{
+		ClientID:     "CLIENT_ID",
+		ClientSecret: "CLIENT_SECRET",
+		Scope:        "SCOPE",
+	}
+
+	sdkConfig := edgecast.NewSDKConfig()
+	sdkConfig.APIToken = apiToken
+	sdkConfig.IDSCredentials = idsCredentials
+
+	originV3Service, err := originv3.New(sdkConfig)
+
+	if err != nil {
+		fmt.Printf("error creating service: %v\n", err)
+		return
+	}
+
+	// Create a Group, or provide a groupID if one exists.
+	originGroupRequest := createOriginGroupRequest()
+	createGroupParams := originv3.NewPostHttpLargeGroupsParams()
+	createGroupParams.CustomerOriginGroupHTTPRequest = originGroupRequest
+	originGroup, err := originV3Service.HttpLargeOnly.PostHttpLargeGroups(
+		createGroupParams,
+	)
+
+	if err != nil {
+		fmt.Printf("error creating origin group: %v\n", err)
+		return
+	}
+
+	fmt.Println("successfully created origin group")
+	fmt.Printf("%# v", pretty.Formatter(originGroup))
+
+	groupid := *originGroup.Id
+
+	fmt.Println("")
+	fmt.Println("**** ADD ORIGIN ENTRY ****")
+	fmt.Println("")
+
+	addParams := originv3.NewAddAdnParams()
+	addParams.MediaType = "http-large"
+
+	origin := originv3.NewCustomerOriginRequest(
+		"cdn-origin-example.com",
+		false,
+		int32(groupid),
+	)
+	addParams.CustomerOriginRequest = *origin
+
+	resp, err := originV3Service.Common.AddAdn(addParams)
```

Comment:
Not sure if this is intended, but the media type is set to HTTP Large and the ADN endpoint is being used.

Reply:
Oh nevermind... this method name is just named badly. Need to re-generate the originv3 code to fix this.